flut_news/lib/screens/main_screen.dart

```diff
@@ -38,17 +38,38 @@ class _MainScreenState extends State<MainScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-
+      /*  appBar: AppBar(
```

Comment:
burdaki açıklama satırı açılacak mı

Reply:
yok açılmayacak

Reply:
eğer beğenmezsen eski haline getirmen zorlamasın diye bıraktım